.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "endOfLine": "auto",
+  "semi": true,
```

Comment:
Does `endOfLine: auto` work okay? On different os you may get different carriages. I've saw `lf` for UNIX platforms